instrumentation/src/main/java/io/opentelemetry/android/OpenTelemetryRumBuilder.java

```diff
@@ -308,7 +314,34 @@ private SdkTracerProvider buildTracerProvider(SessionId sessionId, Application a
     private SpanExporter buildSpanExporter() {
         // TODO: Default to otlp...but how can we make endpoint and auth mandatory?
         SpanExporter defaultExporter = LoggingSpanExporter.create();
-        return spanExporterCustomizer.apply(defaultExporter);
+        SpanExporter spanExporter;
```

Comment:
```suggestion
        SpanExporter spanExporter = defaultExporter;
```

If you do this assignment early here, you can remove the assignment in the `catch` block and the entire `else` block.